README.md

```diff
@@ -36,12 +36,16 @@ Contains Protocol Buffer files describing messages sent to/received from the ARI
 - **reorder-frame** - sample program/unit test that illustrates reordering an ARIS frame; this makes use of the reordering code in aris-integration-sdk/common/code/.
 - **vc-using-framestream** - sample program that implements a full range of commands, records frames to a file, and, optionally, uses multicast for frame acquisition.
 
-### Tools
+### ARIS Tools
 
 **[aris-integration-sdk/tools](tools/)**: contains the following:
 
 - **arislog** - this Windows console application displays messages received from syslog relays, which may be useful for debugging client code. Note that a successful connection must be made after power on to set up forwarding of ARIS syslog messages to the client PC. Note also that this program will display **all** syslog messages forwarded to the PC, even from other applications.
 
+### External Tools
+
+This repository now makes use of **vcpkg** for building **arislog** and **vc-using-framestream**. **vcpkg** is an external tool that cannot be pinned to a specific version when fetching a library, so there is a slight chance you may find an issue when building these files. That said, our daily build should catch any issue pretty quickly.
```

Comment:
Thanks, Robert!

> Each commit in Vcpkg corresponds to a SHA512-hash-checked set of libraries

I was not aware of this. If by "libraries" you mean the downloadable packages, such as protobuf or openssl, then I think I just need to correct our README. (Okay, I'm looking through the ports folder, I think I get it now.) We currently have vcpkg as a submodule. To clarify:

So long as we don't update our vcpkg submodule or run `vcpkg update` the protobuf package that we use via vcpkg will remain at the current package version--**is this correct?**

I did look at export... the protobuf debug libs were big enough to force us into LFS.